### PR TITLE
Add app store link to EditLocationDialog

### LINKED ISSUE #4298: [CAL-101] Add/Edit location dialog: "can't find the right video app? visit our app store"
link with target="_blank" to app store here: 

"Can't find the right video app? Visit our `<a target="_blank" href="/apps/categories/video">App Store</a>`"

make sure to use i18n correctly: https://react.i18next.com/latest/trans-component#using-with-react-components

CleanShot 2022-09-08 at 17 21 18@2x

### PR DESCRIPTION
**Pull request by john-afolabi**
## What does this PR do?

Adds "can't find the right video app? visit our app store" to EditLocationDialog 

Fixes #4298 

**Environment**: Staging(main branch) / Production

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

**Comment by sean-brydon** [comment]
@john-afolabi  looks great - nice job! 

Only thing is this should be in the V2 component for this file (Our fault we didn't mention it on the ticket) 

Would you be able to remove the change from the 

V1:
`apps/web/components/dialog/EditLocationDialog.tsx`

and add it to the V2 file:
`apps/web/components/v2/eventtype/EditLocationDialog.tsx`

**Review by sean-brydon — CHANGES_REQUESTED** [review]
See comment

**Comment by john-afolabi** [comment]
> See comment



> @john-afolabi looks great - nice job!
> 
> Only thing is this should be in the V2 component for this file (Our fault we didn't mention it on the ticket)
> 
> Would you be able to remove the change from the
> 
> V1: `apps/web/components/dialog/EditLocationDialog.tsx`
> 
> and add it to the V2 file: `apps/web/components/v2/eventtype/EditLocationDialog.tsx`

Done, thank you!

File: apps/web/public/static/locales/en/common.json [hunk 1]
```diff
@@ -685,6 +685,7 @@
   "event_type_updated_successfully_description": "Your event type has been updated successfully.",
   "hide_event_type": "Hide event type",
   "edit_location": "Edit location",
+  "cant_find_the_right_video_app": "Can't find the right video app? Visit our",
```
Comment (hariombalhara) [inline apps/web/public/static/locales/en/common.json:688]: Maybe see how `broken_video_action` is implemented. 
"Can't find the right video app? Visit our App Store" should be a single sentence for translation. Breaking and concatenating strings after translation can have completely wrong meaning in some languages.

**Review by hariombalhara — CHANGES_REQUESTED** [review]

**Comment by PeerRich** [comment]
closed due to inactivity and unaddressed requested changes

**Pull request closed without merging**